src/test/unit/ilp.test.js

```diff
@@ -51,19 +56,19 @@ describe('ILP', () => {
 
 
     test('ILP fulfilment should match condition', () => {
-        const {fulfilment, ilpPacket, condition} = ilp.getQuoteResponseIlp(quoteRequest, partialResponse);
+        const {fulfilment, condition} = ilp.getQuoteResponseIlp(quoteRequest, partialResponse);
 
-        const binaryPacket = Buffer.from(ilpPacket, 'base64');
-        const jsonPacket = IlpPacket.deserializeIlpPacket(binaryPacket);
-        console.log(`Decoded ILP packet: ${util.inspect(jsonPacket)}`);
+        // const binaryPacket = Buffer.from(ilpPacket, 'base64');
```

Comment:
why this test is commented out?

Reply:
I was removing lines that aren't used when I removed the comments. I think perhaps this test needs some fixing.

Reply:
was this test working before?